Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListViewModel.swift

```diff
@@ -217,13 +217,17 @@ open class ChatChannelListViewModel: ObservableObject, ChatChannelListController
         controller.deleteChannel { [weak self] error in
             if error != nil {
                 // handle error
-                self?.channelAlertType = .error
+                self?.setChannelAlertType(.error)
```

Comment:
Does the customer have access to the actual error? 🤔 Otherwise, if they want to display an alert depending on the error type, it will be difficult. Make this more flexible, we should have at least in the future something like we do on the ThreadList, where a customer can totaly customize the Error view by implementing a modifier: https://getstream.slack.com/archives/C07T4QV019S/p1741629063926189?thread_ts=1741627501.125659&cid=C07T4QV019S

Reply:
yes, we can expose a modifier in the future. Basically this customer wants to disable all errors.